lib/features/profiles/screens/own_profile_screen.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:gruene_app/features/profiles/widgets/profile_base_data_widget.dart';
+import 'package:gruene_app/features/profiles/widgets/profile_box.dart';
+import 'package:gruene_app/features/profiles/widgets/profile_box_item.dart';
+import 'package:gruene_app/features/profiles/widgets/profile_header_widget.dart';
+import 'package:gruene_app/i18n/translations.g.dart';
+import 'package:gruene_app/swagger_generated_code/gruene_api.swagger.dart';
+
+class OwnProfileScreen extends StatelessWidget {
+  const OwnProfileScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureLoadingScreen(
+      load: fetchOwnProfile,
+      buildChild: (Profile? data) {
+        if (data == null) {
+          return ErrorScreen(error: t.profiles.noResult, retry: fetchOwnProfile);
+        }
+
+        Iterable<ProfileRole> mandateRoles =
+            data.roles.where((role) => [ProfileRoleType.mandate, ProfileRoleType.office].contains(role.type));
+        Iterable<ProfileRole> sherpaRoles = data.roles.where((role) => role.type == ProfileRoleType.role);
+        Iterable<ProfileTag> skillTags = data.tags.where((tag) => tag.type == ProfileTagType.skill);
+        DivisionMembership? kvMembership =
+            data.memberships?.firstWhere((membership) => membership.division.level == DivisionLevel.kv);
+
+        return ListView(
+          children: [
+            SizedBox(height: 24),
+            ProfileHeaderWidget(profile: data),
+            SizedBox(height: 24),
+            ProfileBaseDataWidget(profile: data),
+            SizedBox(height: 12),
+            if (data.memberships!.isNotEmpty) ...[
+              ProfileBox(
+                title: t.profiles.memberships,
+                items: data.memberships!
+                    .map(
+                      (membership) =>
+                          ProfileBoxItem(title: '${membership.division.name1} ${membership.division.name2}'),
+                    )
+                    .toList(),
+              ),
+              SizedBox(height: 12),
+            ],
+            if (mandateRoles.isNotEmpty) ...[
+              ProfileBox(
+                title: t.profiles.mandates,
+                items: mandateRoles.map((role) => ProfileBoxItem(title: role.alias)).toList(),
+              ),
+              SizedBox(height: 12),
+            ],
+            if (sherpaRoles.isNotEmpty) ...[
+              ProfileBox(
+                title: t.profiles.sherpaRole,
+                items: sherpaRoles.map((role) => ProfileBoxItem(title: role.alias)).toList(),
+              ),
+              SizedBox(height: 12),
+            ],
+            if (skillTags.isNotEmpty) ...[
+              ProfileBox(
+                title: t.profiles.skills,
+                items: skillTags.map((tag) => ProfileBoxItem(title: tag.label)).toList(),
+              ),
+              SizedBox(height: 12),
+            ],
+            if (kvMembership!.division.urls.isNotEmpty) ...[
```

Comment:
:question: Same question as above, are we guaranteed that everybody has a `kvMembership`?